paper-icon-button.html

```diff
@@ -76,7 +76,11 @@
         position: relative;
         padding: 8px;
         outline: none;
-        -webkit-tap-highlight-color: rgba(0,0,0,0);
+
+        /* NOTE: Both values are needed, since some phones require the value to be `transparent`. */
```

Comment:
Sincerely, I'm not sure... In a quick search: http://phonegap-tips.com/articles/essential-phonegap-css-webkit-tap-highlight-color.html#comment-904717757